tests/bearlib/aspects/ClassTest.py

```diff
@@ -31,3 +33,10 @@ class docs:
                 example = 'Example'
 
         assert not SubAspect.docs.check_consistency()
+        assert issubaspect(SubAspect, RootAspect)
+        assert not issubaspect(Root, RootAspect)
+        assert not issubaspect(RootAspect, RootAspect)
+        with pytest.raises(TypeError):
+            issubaspect('String', SubAspect)
+        with pytest.raises(TypeError):
+            issubaspect(RootAspect, str)
```

Comment:
And as @Techievena mentioned in https://github.com/coala/coala/pull/3996#discussion_r108152329 the tests for aspectclass instances are still missing...